tests/test_search_rollup.py

```diff
@@ -8,7 +8,7 @@
 from pyspark.sql.types import StructType, StructField, StringType, ArrayType, \
     LongType
 
-from mozetl.search import daily_search_rollups as search_rollups
+from mozetl.search import search_rollups
```

Comment:
We'll need a corresponding PR over in `telemetry-airflow` to run this using the new name:
https://github.com/mozilla/telemetry-airflow/blob/master/jobs/search_rollups.sh#L9